qa/suites/fs/upgrade/nofs/branch/pacific.yaml

```diff
@@ -0,0 +1,2 @@
+teuthology:
+  branch: pacific
```

Comment:
This isn't required. We should keep q->s and r->s. Right now it's just p->r (and not even q->r, but I think there is value is adding the N-1 release upgrade).

Reply:
OK. So, a thing about nofs - it was introduces to test upgrades without a file system since there was a bug which used to crash the mons when no fs was present and an upgrade was done. However, it has always tested upgrading to the latest branch sha, until:

```
commit 66fd89a860e3a3258849105d7f9149f5baf804b8
Author: Dhairya Parmar <dparmar@redhat.com>
Date:   Sun Oct 1 03:13:18 2023 +0530

    qa/upgrade/nofs: upgrade pacific->reef

    Fixes: https://tracker.ceph.com/issues/62953
    Signed-off-by: Dhairya Parmar <dparmar@redhat.com>

diff --git a/qa/suites/fs/upgrade/nofs/tasks/1-upgrade.yaml b/qa/suites/fs/upgrade/nofs/tasks/1-upgrade.yaml
index 85814287138..e31e4b43ca8 100644
--- a/qa/suites/fs/upgrade/nofs/tasks/1-upgrade.yaml
+++ b/qa/suites/fs/upgrade/nofs/tasks/1-upgrade.yaml
@@ -23,6 +23,7 @@ tasks:
     - ceph fs dump
 - install.upgrade:
     mon.a:
+      branch: reef
 - print: "**** done install.upgrade"
 - ceph.restart:
     daemons: [mon.*, mgr.*]
```

which introduce p->r upgrades. So, we need to change this too upgrade from N-1/N-2 (r/s) to latest sha. Agreed?

Reply:
okay, from the conversations today i think we can't just have n-1|n-2 in all sub-suites (i.e. we need to have sub-suite specific changes and not generalise YAMLs(?)) so that should mean https://github.com/ceph/ceph/pull/56476#issuecomment-2060514326 is nullified (?). 

Let us create a list to summarise what needs to be done for each sub-suite, there are total five sub-suites

1. `featureful_client/old_client`: only octopus -> reef
2. `featureful_client/upgraded_client`: ?
3. `mds_upgrade_sequence`: this doesn't do src->dest type of upgrades, quincy always existed so unsure if this should have n-1|n-2
4. `nofs`: n-1|n-2 under discussion
5. `upgraded_client` - n-1|n-2 under discussion

please update this if something's wrong

Reply:
yeah, and pacific is now EOL too so makes sense to upgrade from n-1|n-2